src/combinatorics/set_family/shadow.lean

```diff
@@ -0,0 +1,131 @@
+/-
+Copyright (c) 2021 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta, Alena Gusakov, Yaël Dillies
+-/
+import data.finset.lattice
+
+/-!
+# Shadows
+
+This file defines shadows of a set family. The shadow of a set family is the set family of sets we
+get by removing any element from any set of the original family. If one pictures `finset α` as a big
+hypercube (each dimension being membership of a given element), then taking the shadow corresponds
+to projecting each finset down once in all available directions.
+
+## Main definitions
+
+The `shadow` of a set family is everything we can get by removing an element from each set.
+
+## Notation
+
+`∂ 𝒜` is notation for `shadow 𝒜`. It is situated in locale `finset_family`.
```

Comment:
I used the mathscr unicode A through this file for finset families to match my notes (I was *very* new to Lean when the first version of this file was made), perhaps more sensible choices should be considered?

Reply:
I personally like that choice. In general, your branch was very nice-looking notation-wise, but the notation was pretty obscure. Another idea would be to use `s` and `t` for the finsets and `A` for the family.

Reply:
Yeah `s` and `t` for the finsets would match nicely with `data/finset` as well

Reply:
Do we want to adjust to `A, B : finset (finset α)` before merging?

Reply:
I quite like it like that. But I can change if you really want to.